django/db/backends/base/base.py

```diff
@@ -37,6 +38,9 @@ class BaseDatabaseWrapper(object):
     vendor = 'unknown'
     SchemaEditorClass = None
 
+    # Classes instantiated inside __init__().
```

Comment:
I'm not sure the comment is adding much value and I'm unsure about the spaces around these sections unlike the other attributes. Also, the naming style difference between `SchemaEditorClass` is a bit unfortunate. I'm not sure which to prefer.